<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsFrame</class>
 <widget class="QFrame" name="GraphicsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="GraphicsView"/>
     </item>
     <item row="1" column="1">
      <widget class="QScrollArea" name="ParametersScrollArea">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="ParametersScrollAreaWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>148</width>
          <height>576</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="ParametersVerticalLayout">
           <property name="spacing">
            <number>10</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="SavePicture_button">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить изображение</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Print_button">
       <property name="text">
        <string>Печать изображения</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Recalculate_button">
     <property name="text">
      <string>Пересчитать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
